src/main.cpp

```diff
@@ -40,6 +40,11 @@ void getServerOptions(NginxConfig config, http::server::server_options *server_o
       server_options_pointer->port = statement.at(1);
     }
 
+    // Threads
+    if (statement.size() == 2 && statement.at(0) == "threads") {
```

Comment:
Maybe we could do some error handling here to make sure the string actually represents an integer